nav-app/src/app/layer-upgrade/layer-upgrade.component.html

```diff
@@ -0,0 +1,191 @@
+                                </div>
+                                <div class="float"><b>{{attackID}}</b></div>
+                                <div class="float">{{getTechnique(attackID, viewModel).name}}</div>
+                            </div>
+                        </mat-panel-title>
+                    </mat-expansion-panel-header>
+
+                    <ng-template matExpansionPanelContent> <!-- technique lazy render -->
+
+                    <!-- version header -->
+                    <div class="cols">
+                        <div class="float wide">
+                            <div class="version">ATT&CK {{compareTo.version}}</div>
+                            <mat-divider></mat-divider>
+                        </div>
+                        <div class="float arrow"><mat-icon inline="true">east</mat-icon></div>
+                        <div class="float wide">
+                            <div class="version">ATT&CK {{viewModel.version}}</div>
+                            <mat-divider></mat-divider>
+                        </div>
+                    </div>
+
+                    <div class="cols" *ngIf="tacticsChanged(attackID, section)">
+                        <!-- technique from previous version -->
+                        <div class="float wide" *ngIf="getTechnique(attackID, compareTo) as oldTechnique">
+                            <div class="changelog-cells" *ngFor="let tactic of getTactics(attackID, compareTo)">
+                                <div [dndDraggable]="tactic.shortname">
+                                    <changelog-cell
+                                        [tactic]="tactic"
+                                        [technique]="oldTechnique" 
+                                        [viewModel]="compareTo"
+                                        [isCurrentVersion]="false">
+                                    </changelog-cell>
+                                </div>
+                            </div>
+                        </div>
+
+                        <!-- drag&drop tooltip info -->
+                        <div class="float arrow">
+                            <mat-icon class="info" matTooltipPosition="above" 
+                                matTooltip='To copy annotations, drag a cell with annotations from ATT&CK {{compareTo.version}} to a cell in ATT&CK {{viewModel.version}}.'>
+                                info_outline
+                            </mat-icon>
+                        </div>
+
+                        <!-- technique from latest version-->
+                        <div class="float wide" *ngIf="getTechnique(attackID, viewModel, section) as newTechnique">
+                            <div class="changelog-cells" *ngFor="let tactic of getTactics(attackID, viewModel, section)"
+                                dndDropzone (dndDrop)="onDrop($event, attackID, newTechnique, tactic)">
```

Comment:
There's actually a console error when you drop to another technique, so this might take some actual implementation to avoid errors even if the UI can't be fixed:

<img width="404" alt="Screen Shot 2021-07-12 at 1 07 17 PM" src="https://user-images.githubusercontent.com/28868888/125332649-4b2e2380-e317-11eb-8d3e-be205fa2a675.png">
.